org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/service/ServiceExtension.java

```diff
@@ -132,7 +133,7 @@ public boolean supportsParameter(ParameterContext parameterContext, ExtensionCon
 			serviceType = ((ParameterizedType) genericMemberType).getActualTypeArguments()[0];
 		}
 		// The service type must be a raw type
-		if (serviceType instanceof Class) {
+		if (serviceType instanceof Class || serviceType instanceof WildcardType) {
```

Comment:
I think we need a dedicated type for this like AnyService in DS. See <https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#org.osgi.service.component.AnyService>.

Reply:
We also need to test that the injection site is type compatible with any possible service object. For example, it is of type Object.

> When using this special service name, the reference field or parameter must be of type java.lang.Object so that any service object can be provided.